test/sharness/t0250-files-api.sh

```diff
@@ -316,13 +316,26 @@ test_files_api() {
 		verify_dir_contents /cats file1 ipfs this
 	'
 
+	test_expect_success "write 'no-flush' succeeds" '
+		echo "testing" | ipfs files write -f -e /cats/walrus
+	'
+
+	test_expect_success "changes bubbled up to root on inspection" '
+		ipfs files stat / | head -n1 > root_hash
```

Comment:
can we test that they _didn't_ bubble up beforehand? are there ops that will read the unsynced thing? also-- it makes me uneasy that sync requires another op hit, no matter how long time passes. i would imagine that `--sync=false` is an optimization for heavy load, but that a `sync` would happen if time passed. (like an os buffer cache)